### PR TITLE
Eliminate unclear descriptions on the 'Setting Deployment Resources' section follow up

### PR DESCRIPTION
**Pull request by mburke5678**
Editorial follow-up to https://github.com/openshift/openshift-docs/pull/25580/

Associated BZ: https://bugzilla.redhat.com/show_bug.cgi?id=1880238

Preview: https://deploy-preview-28692--osdocs.netlify.app/openshift-enterprise/latest/dev_guide/deployments/basic_deployment_operations.html#deployment-resources

### CODE REVIEW

**Comment by mburke5678** [comment]
@bysnupy Would you mind taking a look at my changes to make sure I didn't change the meaning of the text? Thank you!

**Comment by mburke5678** [comment]
@xltian Can you please assign this PR for QE review? Thank you in advance.

**Comment by bysnupy** [comment]
@mburke5678 It's LGTM ! Thanks !

**Comment by sunilcio** [comment]
@mburke5678 LGTM, thanks!

**Review by kalexand-rh — COMMENTED** [review]
I have one word usage item to ponder, but this otherwise LGTM.

File: dev_guide/deployments/basic_deployment_operations.adoc [hunk 1]
```diff
@@ -291,18 +291,17 @@ $ oc set triggers --help
 [[deployment-resources]]
 == Setting Deployment Resources
 
+A deployment is completed by a deployment pod. By default, a deployment pod consumes unbounded node resources on the compute node where it is scheduled. In most cases, the unbound resource consumption does not cause a problem, because deployment pods consume low resources and run for a short period of time. If a project specifies default container limits, the resources consumed by a deployment pod, along with any other pods, count against those limits.
```
[the remark this answers is not in the document]
Comment (mburke5678) [reply dev_guide/deployments/basic_deployment_operations.adoc:294]: That word was in the docs previously. I can change to _use_.

**Pull request merged**